### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by MiaoZhangAWS**
Add a pull request template for our repository so that we can standardize the pull request information. 
Following [this instruction](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) to create a pull request template. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by KareemAliAWS — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Motivation
+- Please give a brief description for the background of this change.
+- Issue number: 
+
+
+### Modifications
+#### Change summary
+Please describe what changes are included in this CR.
```
Comment (HarshGandhi-AWS) [inline .github/pull_request_template.md:8]: Can you update CR to code change or change request? Everything else looks good.

Comment (MiaoZhangAWS) [reply .github/pull_request_template.md]: Changed.

**Review by HarshGandhi-AWS — APPROVED** [review]
LGTM

**Review by KareemAliAWS — APPROVED** [review]

**Comment by MiaoZhangAWS** [comment]
Close this request as current pull request does not work well on a forked branch.

**Pull request closed without merging**